skills/sap-btp-cias/references/setup-guide.md

```
@@ -6,6 +6,36 @@ Complete guide for subscribing to and configuring Cloud Integration Automation S
 
 ---
 
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Subscribe to CIAS - Standard Plan](#subscribe-to-cias---standard-plan)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)